compute/k8s-services/vars.tf

```diff
@@ -67,6 +67,11 @@ variable "cloudwatchlogs_iam_role_deploy" {
 # Traefik
 # --------------------------------------------------
 
+variable "traefik_svc_name" {
+  type    = string
+  default = "traefik"
```

Comment:
Unless we are explicit about service names, this will clash with multiple instances. Consider setting service name to just deploy_name, and not have a separate variable?

Or default to null, and configure the `kubernetes_service` resource to use supplied service name, or fallback to deploy_name if null?

Reply:
I am not sure I understand this suggestion. The traefik_deploy module was already hardcoding "traefik" as deploy_name. So this was partially to avoid hardcoding, but more importantly to also use the variable traefik_svc_name in conditional statements to determine locals defined in dependencies.tf

Reply:
Also keep in mind that there is no such variable called deploy_name in k8s-services, so the traefik_svc_name variable is passed down to the k8s-traefik sub module's deploy_name variable.

Reply:
Added documentation for the "weird" logic on how to calculate the traefik dashboard hostname. And also remove the traefik_svc_name variable.